src/aliceVision/sfmDataIO/AlembicImporter.cpp

```diff
@@ -329,7 +329,7 @@ bool readPointCloud(const Version & abcVersion, IObject iObj, M44d mat, sfmData:
 
     }
     else {
-        ALICEVISION_LOG_WARNING("Alembic LOAD: NO OBSERVATIONS_WITH_FEATURES: "
+        ALICEVISION_LOG_WARNING("Alembic hb: NO OBSERVATIONS_WITH_FEATURES: "
```

Comment:
?

Reply:
seems to be still there in the github global diff

Reply:
...